static/info2016.html

```diff
@@ -0,0 +1,303 @@
+                        <td>Lunch</td>
+                    </tr>
+                    <tr>
+                        <td>14:00-16:00</td>
+                        <td>Sessions</td>
+                    </tr>
+                    <tr>
+                        <td>16:00-17:00</td>
+                        <td>Review & Closing</td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </div>
+</div>
+
+
+<div class="container">
+    <div class="row">
+        <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDr5F59SnCVuS2CjZqWGvuZDy4JnlkdXp0"></script>
+    </div>
+</div>
+
+
+<a name="location"></a>
+<h4 class="header center blue-text">Location</h4>
+<div id="map_wrapper">
+    <div id="map"></div>
+    <div id="map_overlay">
+        <img src="img/vi_logo.jpeg" alt="Virtual Identity AG" width="120px">
+        <h5>Virtual Identity AG</h5>
+        <p class="lead">
+            Isarwinkel 16<br>
+            81379 München<br>
+            <a href="http://virtual-identity.com/">http://virtual-identity.com</a>
+        </p>
+        <p class="lead">
+            by U-Bahn: Thalkirchen (U3), 5 minute walk<br>
+            by Car: There are parking areas nearby
+        </p>
+    </div>
+</div>
+<footer>
+  <div>
+    <h3 class="colorOfTheYear-text">Sitemap</h3>
+    <ul>
+    
+        
+        <li><a class="colorOfTheYear-text" href="/">Home</a></li>
+    
+        
+        <li><a class="colorOfTheYear-text" href="https://github.com/jscraftcamp/website/projects/10">Sponsoring</a></li>
+    
+        
+        <li><a class="colorOfTheYear-text" href="/values.html">Values</a></li>
+    
+    </ul>
+  </div>
+  <div>
+    <h3 class="colorOfTheYear-text">Get in touch</h3>
+    <ul>
+      <li><a class="colorOfTheYear-text" href="/theteam.html">The Team</a></li>
+      <li><a class="colorOfTheYear-text" href="https://twitter.com/jscraftcamp">Twitter</a></li>
+      <li><a class="colorOfTheYear-text" href="/imprint.html">Imprint</a></li>
+      <li><a class="colorOfTheYear-text" href="/privacypolicy.html">Privacy Policy</a></li>
+    </ul>
+  </div>
+  <div style="flex: 1.5">
+    <h3 class="colorOfTheYear-text">Archive</h3>
+    <ul>
+      
+          
+          <li><a class="colorOfTheYear-text" href="/info2019">2019 - Home</a></li>
+      
+          
+          <li><a class="colorOfTheYear-text" href="/info2018">2018 - Home</a></li>
+      
+          
+          <li><a class="colorOfTheYear-text" href="/photos2018">2018 - Photos</a></li>
+      
+          
+          <li><a class="colorOfTheYear-text" href="/info2017">2017 - Home</a></li>
+      
+          
+          <li><a class="colorOfTheYear-text" href="/photos2017">2017 - Photos</a></li>
+      
+          
+          <li><a class="colorOfTheYear-text" href="/info2016">2016 - Home</a></li>
+      
+          
+          <li><a class="colorOfTheYear-text" href="/photos2016">2016 - Photos</a></li>
```

Comment:
All those links are not working anymore, as they are missing a .html in the end (e.g. `/photos2016` should actually be `/photos2016.html`).

Just a minor thing I want to point out here - I don't think we need to act on this as it is too much work for now.
I'll open a discussion issue if we even can delete all those old pages.

Reply:
good news, @davelosert :
I tested it, to see what links are wrong and how much we must change, but:
👍 gatsby has some redirects built in!
e.g.
Both links are served containing the same page:

* https://deploy-preview-843--jscraftcamp.netlify.com/info2016
* https://deploy-preview-843--jscraftcamp.netlify.com/info2016.html

So, we do not need to change anything 👍

Reply:
... BUT while local development we have issues, because there 
is no automatic `/info2016` route for the `/info2016.html` page